lib/string.c

```diff
@@ -193,6 +193,7 @@ long strtol(const char *nptr, char **endptr, int base)
     return (acc);
 }
 
+#ifndef KTF_ACPICA
```

Comment:
Why is this conditional?

Reply:
ACPICA comes with its own implementation of these (which is preferable I suppose).